src/config.json

```diff
@@ -2,5 +2,8 @@
   "redText": "Hard problem",
   "orangeText": "",
   "greenText": "Controversial",
-  "blueText": "Maybe check later?"
+  "blueText": "Maybe check later?",
```

Comment:
It might make sense to have all defaults `""` now that we're pulling in text from the `FlagManager` potentially

Reply:
that makes sense